src/DependencyInjection/Configuration.php

```diff
@@ -629,10 +629,10 @@ private function addSharedPluginNodes(ArrayNodeDefinition $pluginNode, $disableA
             ->canBeEnabled()
             ->addDefaultsIfNotSet()
             ->children()
-                ->scalarNode('name')->end()
-                ->scalarNode('key')->defaultNull()->end()
-                ->integerNode('tokens')->defaultValue(1)->end()
-                ->floatNode('max_time')->defaultNull()->end()
+                ->scalarNode('name')->isRequired()->info('Rate limiter configuration name from rate_limiter.yaml')->end()
+                ->scalarNode('key')->defaultNull()->info('A unique key for using one rate limiter name for different clients')->end()
```

Comment:
i just read up on rate limiter, now i see what it does. we could also default this to the client name instead of null, not sure which way is more expected. like this we err on the side of limiting too much if the user does not understand the documentation, so i think its the safe default.

i suggest adding a bit more to the info text:

```suggestion
                ->scalarNode('key')->defaultNull()->info('Key to avoid sharing this rate limiter with other clients or other services. You can use the name of the client for example.')->end()
```

Reply:
what i meant is we could use the name of the http client where the plugin is attached as key for the rate limiter, so the limit applies per client instead of globally for the limiter.

but that was thinking out loud, lets keep as is and only adjust the documentation with my suggestion if you agree